### PR TITLE
E UI e UI 3846 change loading on details page

### PR DESCRIPTION
**Pull request by waynepHMCTS**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3846

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by tarikdogan-hmcts — APPROVED** [review]

File: src/org-manager/components/new-pbas-info/new-pbas-info.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h1 class="govuk-heading-xl">Approve new PBA number</h1>
   <h3 class="govuk-heading-m">PBA accounts</h3>
 
   <app-pba-account-approval *ngFor="let pba of org.pbaNumber; let i = index" [pbaNumbers]="org.pbaNumber.join(',')"
-    [pbaNumber]="pba" [accountName]="org.accountDetails[i]?.account_name" [organisationId]="org.organisationId"
+    [pbaNumber]="pba" [accountName]="org.accountDetails[i] ? org.accountDetails[i].account_name : ''" [organisationId]="org ? org.organisationId : ''"
```
Comment (paulhowes-HMCTS) [inline src/org-manager/components/new-pbas-info/new-pbas-info.component.html:13]: Any particular reason you've chosen the verbose route?

File: src/org-manager/containers/edit-details/edit-details.component.ts [hunk 2]
```diff
@@ -25,8 +24,17 @@ export class EditDetailsComponent implements OnInit, OnDestroy {
   public pbaNumbers: string[];
   public saveDisabled = true;
   public serverError$: Observable<{ type: string; message: string }>;
+  public routeSubscripton: Subscription;
+  public pipeScribtion: Subscription;
```
Comment (paulhowes-HMCTS) [inline src/org-manager/containers/edit-details/edit-details.component.ts:28]: Typo on pipeScribtion

File: src/org-manager/containers/edit-details/edit-details.component.ts [hunk 3]
```diff
@@ -25,8 +24,17 @@ export class EditDetailsComponent implements OnInit, OnDestroy {
...
 
-  constructor(private readonly store: Store<fromStore.OrganisationRootState>) {}
+  constructor(
+    private readonly store: Store<fromStore.OrganisationRootState>,
```
Comment (paulhowes-HMCTS) [inline src/org-manager/containers/edit-details/edit-details.component.ts:31]: Can store be removed now we are calling direct endpoints from the service?

File: src/org-manager/containers/edit-details/edit-details.component.ts [hunk 4]
```diff
@@ -37,17 +45,17 @@ export class EditDetailsComponent implements OnInit, OnDestroy {
...
-            this.store.dispatch(new fromStore.LoadPendingOrganisations());
-          }
-        }));
+    this.orgDetails$ = this.organisationService.getSingleOrganisation({ id: this.orgId })
+      .pipe(map(apiOrg => AppUtils.mapOrganisation(apiOrg)), tap(org => {
+        if (org) {
+          this.orgId = org.organisationId;
+          this.pbaNumbers = org.pbaNumber;
+          this.saveDisabled = !org.pbaNumber;
+        } else if (!org && !this.orgId) {
```
Comment (paulhowes-HMCTS) [inline src/org-manager/containers/edit-details/edit-details.component.ts:54]: We shouldn't be loading from the store anymore

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
I believe the store still needs removing, although I'm happy to discuss it further if that's not the case.

File: src/org-manager/containers/edit-details/edit-details.component.spec.ts [hunk 5]
```diff
@@ -1,47 +1,98 @@
...
-import * as fromRoot from '../../../app/store';
-import * as fromOrganisationPendingStore from '../../store';
+import { of } from 'rxjs';
+import { OrganisationService, PbaAccountDetails, UpdatePbaServices } from 'src/org-manager/services';
 import { EditDetailsComponent } from './edit-details.component';
 
-
-
-describe('EditDetailsComponent', () => {
+fdescribe('EditDetailsComponent', () => {
```
Comment (kuda1992) [inline src/org-manager/containers/edit-details/edit-details.component.spec.ts:12]: Remove fdescribe

**Review by kuda1992 — DISMISSED** [review]
Looks good

**Pull request closed without merging**